templates/base/packages/nextjs/public/manifest.json.template.mjs

```diff
@@ -0,0 +1,15 @@
+import { stringify, withDefaults } from '../../../../utils.js'
+
+const contents = ({ name, description, iconPath, extraContent }) => stringify({
+  name: name[0],
+  description: description[0],
+  ...(iconPath[0] && { iconPath: iconPath[0] }), // Only include iconPath if it's provided
+  ...extraContent[0]
```

Comment:
Why do we need separate iconPath variable can't it be provided inside `extraContent`?